src/Core/Prefab/GlobeView.js

```diff
@@ -399,15 +426,24 @@ GlobeView.prototype.getPickingPositionFromDepth = function getPickingPositionFro
     const prev = this.camera.camera3D.layers.mask;
     this.camera.camera3D.layers.mask = 1 << this.wgs84TileLayer.threejsLayer;
 
-    const previousRenderState = this._renderState;
-    this.changeRenderState(RendererConstant.DEPTH);
-
-    // Render to buffer
-    var buffer = this.mainLoop.gfxEngine.renderViewTobuffer(
-        this,
-        this.mainLoop.gfxEngine.fullSizeRenderTarget,
-        mouse.x, dim.y - mouse.y,
-        1, 1);
+    let buffer;
+    if (viewPaused) {
```

Comment:
Can't you simplify this code? Or extract this to a utility method maybe?

Reply:
ok

Reply:
Thanks, it's much more readable.